SourceryTests/Stub/JavaScriptTemplates/Includes.js

```diff
@@ -0,0 +1 @@
+<%- include('Equality.js') %>-%><%- include('Other.js') %>-%>
```

Comment:
Yes, that's ok, though it still feels like undocumented feature, in all examples on github it was using `<% -%>`. But if it works - fine.
Another thing is that ejs as far as I can tell does not require file extension in include. I think we could do the same.